### PR TITLE
Update installation docs

### PR DESCRIPTION
**Pull request by orangejulius**
We have made progress on making importers more consistent!

Metro extracts has also gotten better!

### CODE REVIEW

File: installing.md [hunk 1]
```diff
@@ -292,13 +283,12 @@ reindex all your data after making schema changes.
 Now that the schema is set up, you're ready to begin importing data!
 
 Our [goal](https://github.com/pelias/pelias/issues/255) is that eventually you'll be able to run all
-the importers with simply `cd $importer_directory; npm start`. Unfortunately only the Who's on First
-and OpenStreetMap importers works that way right now.
+the importers with simply `cd $importer_directory; npm start`. We are now really close, and all but
+one importer follows this pattern!
```
Comment (kkowalsky) [inline installing.md]: let's not use the phrase 'really close', it's too vague.

Comment (orangejulius) [inline installing.md]: Good idea. I expanded on your changes a bit, let me know how they look.

Comment (kkowalsky) [inline installing.md]: I'm being finicky, but don't like describing it changes as "special", as it can be vague

Comment (orangejulius) [inline installing.md]: makes sense. How about now?

**Comment by kkowalsky** [comment]
LGTM (thanks @orangejulius!)

**Comment by trescube** [comment]
👍

**Pull request merged**